lib/app/routes/pages.dart

```diff
@@ -11,6 +11,10 @@ enum Pages {
     path: '/generosity-challenge',
     name: 'GENEROSITY-CHALLENGE',
   ),
+  generositySelectValues(
+    path: 'select-values',
```

Comment:
let's add "generosity-..." for consistency

Reply:
I would reather remove generosity from the title to make it  selectValues(path: 'select-values',) since it is on the generosity sub-route.

Reply:
makes sense bc of sub route, I didn't think about it)